core/arch/arm/kernel/abort.c

```diff
@@ -66,13 +76,13 @@ static void __print_stack_unwind_arm32(struct abort_info *ai)
 
 		utc = to_user_ta_ctx(s->ctx);
 		exidx = utc->exidx_start;
+		/* Here exidx will be NULL if the TA has no unwind tables */
+		if (exidx)
+			exidx += utc->load_addr;
 		exidx_sz = utc->exidx_size;
-		if (!exidx || !exidx_sz) {
-			/* Can't unwind, the PC is as much as we can show */
-			EMSG_RAW(" 0x%08x", ai->pc);
-			return;
-		}
-		exidx += utc->load_addr;
+	} else {
+		exidx = (vaddr_t)&__exidx_start;
```

Comment:
Since `__exidx_start` is declared as an array there's no need to use the `&` also (btw, applying `&` on arrays is often a bad idea).
I wonder if we shouldn't have a special .h file that contains all symbols exported by the linker script, but that's out of scope for this PR.